scheme/arm-cca/store_handler.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package arm_cca
+
+import (
+	"fmt"
+
+	"github.com/veraison/ccatoken"
+	"github.com/veraison/services/handler"
+	"github.com/veraison/services/proto"
+	"github.com/veraison/services/scheme/common/arm"
+	realm "github.com/veraison/services/scheme/common/cca/realm"
+)
+
+type StoreHandler struct{}
+
+func (s StoreHandler) GetName() string {
+	return "cca-store-handler"
+}
+
+func (s StoreHandler) GetAttestationScheme() string {
+	return SchemeName
+}
+
+func (s StoreHandler) GetSupportedMediaTypes() []string {
+	return nil
+}
+
+func (s StoreHandler) SynthKeysFromRefValue(
+	tenantID string,
+	refVal *handler.Endorsement,
+) ([]string, error) {
+	switch refVal.SubType {
+	case "sw-component", "platform-config":
+		return arm.SynthKeysForPlatform(SchemeName, tenantID, refVal)
+	case "realm.reference-value":
```

Comment:
naming is a bit wild here:

* non-namespaced: `sw-component`
* namespaced with `-` separator: `platform-config`
* namespaced with `.` separator: `realm.reference-value`

:-)

probably worth fixing?

Reply:
Thinking a bit more I would change it to following:
Platform SW Components:
`platform.sw-component`

Platform Configuration:
`platform.config`

Realm Reference Value:

`realm.reference-value`